### DISCUSSION TITLE
Version 4.2.2 identifiée comme version de développement

### DISCUSSION THREAD

**Discussion by LucN31** [JeromeDevome/GRR#319] [General]
Bonsoir,
Depuis la mise à jour vers la dernière version (4.2.2), le tableau de bord indique un warning disant « *version de développement, ne pas utiliser en production* ». 
Comment retirer ce warning ?
Merci.

**Comment by JeromeDevome** [comment]
Bonjour, 
Avez vous téléchargé depuis la branche 4.2 ou la page https://github.com/JeromeDevome/GRR/releases ?
